psi4/src/psi4/libqt/qt.h

```diff
@@ -302,6 +305,7 @@ int C_DORMLQ(char side, char trans, int m, int n, int k, double* a, int lda, dou
              double* work, int lwork);
 int C_DORMQL(char side, char trans, int m, int n, int k, double* a, int lda, double* tau, double* c, int ldc,
              double* work, int lwork);
+PSI_API
```

Comment:
I don't object to these `PSI_API`s, but I would think they'd only be needed for plugins, not for SAPT(DFT) that's all internal to psi. Just curious why added.

Reply:
I was testing for these LAPACK subroutines in a plugin because I needed to figure out what is the correct format of arguments (since these are in the Fortran column-major format).